package.json

```
@@ -2,8 +2,9 @@
   "repository": "git@github.com:galaniprojects/ctypehub.git",
   "license": "BSD-4-Clause",
   "type": "module",
+  "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e",
```

I assume this line was added by corepack automatically. We can make it shorter:
```suggestion
  "packageManager": "pnpm@9.6.0",
```